src/to_square_example.c

```diff
@@ -0,0 +1,177 @@
+#include <pebble.h>
+
+
+#define ABS(a) (((a) > 0) ? (a) : -1 * (a))
+#define IMAGES_LENGTH 2
+
+
+
+static Window *s_main_window;
+static Layer *icon_layer;
```

Comment:
Keep up the style of `s_name` as you have done with the `Window` variable for all other `static` variables.